.golangci.yml

```diff
@@ -0,0 +1,37 @@
+linters-settings:
+  gocyclo:
+    min-complexity: 40 # for dashboard ReconcileComponent()
```

Comment:
Perhaps it would be better to set nolint for this component instead. 40 is quite a high number to keep it as a general rule.

Reply:
good idea, i will fix it